spec/lib/api/reminders_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
+require 'uber'
+
+describe Uber::API::Reminders do
+  let!(:client) { setup_client }
+
+  describe "get reminder" do
+    before do
+      # From: https://developer.uber.com/docs/rides/api/v1-reminders-get
+      stub_uber_request(:get, 'v1/reminders/48ee034e-311d-41ea-9794-02fc8dcd8696',
+                        {"reminder_status"=>"pending", "reminder_id"=>"48ee034e-311d-41ea-9794-02fc8dcd8696", "reminder_time"=>1473296851,
+                         "event"=>{"latitude"=>nil, "time"=>1473507651, "name"=>nil, "longitude"=>nil, "location"=>nil}, "product_id"=>{}}
+      )
+    end
+
+    it "should return details about reminder" do
+      reminder = client.reminder('48ee034e-311d-41ea-9794-02fc8dcd8696')
+      expect(reminder).to be_instance_of Uber::Reminder
+      expect(reminder.event).to be_instance_of Uber::Reminder::Event
+      expect(reminder.reminder_id).to eq "48ee034e-311d-41ea-9794-02fc8dcd8696"
+    end
+
+    it "should also respond to reminder_detail" do
+      reminder = client.reminder_detail('48ee034e-311d-41ea-9794-02fc8dcd8696')
+      expect(reminder).to be_instance_of Uber::Reminder
+      expect(reminder.event).to be_instance_of Uber::Reminder::Event
+      expect(reminder.reminder_id).to eq "48ee034e-311d-41ea-9794-02fc8dcd8696"
+    end
+  end
+
+  describe "add reminder" do
+    before do
+      # From: https://developer.uber.com/docs/rides/api/v1-reminders-post
+      @time_now = Time.at(1473275408)
+      stub_uber_request(:post, 'v1/reminders',
+                        {
+                            reminder_time: 1473357203,
+                            phone_number: '+91-9999999999',
+                            event: {time: 1473509642},
+                            trip_branding: {link_text: 'My first reminder'},
+                            reminder_id: 'rem1'
+                        },
+                        :body => {
```

Comment:
This response body is missing `product_id` and `reminder_status`, do you think we need to test for that as well? :)